homeassistant/components/amcrest/camera.py

```diff
@@ -547,3 +600,22 @@ def _start_tour(self, start):
             log_update_error(
                 _LOGGER, "start" if start else "stop", self.name, "camera tour", error
             )
+
+    def _ptz_control(self, movement, travl_time):
+        """Move or zoom camera in specified direction."""
+        code = _ACTION[_MOV.index(movement)]
+        try:
+            channel = 1
+            hspeed = 0
+            vspeed = "vertical_speed=1"
+            if "zoom" in movement:
+                vspeed = 0
+
+            self._api.ptz_control_command(channel, "start", code, hspeed, vspeed)
```

Comment:
`channel` should be `0`. Also, I think you need to specify `arg3`, otherwise it will result in `&arg3=None`, when it should be `&arg3=0`.

I would just do it this way:
```python
            self._api.ptz_control_command(action="start", code=code, arg1=arg1, arg2=arg2, arg3=0)
```